src/_includes/base.njk

```diff
@@ -32,7 +32,7 @@
 		{% else %}
 		<!-- else mode -->
 		{% endif %}
-		<h1 class="header">{{ title }}</h1>
+		<h1 class="header">{{ title or "NO TITLE" }}</h1>
```

Comment:
Same as above. One template has `undefined` title which threw an error so I added a silly default to get it building.